### DISCUSSION TITLE
Is there a way to programmatically call openWalletModal?

### DISCUSSION THREAD

**Discussion by jcheese1** [rainbow-me/rainbowkit#320] [Q&A] [answered]
title

**Comment by emzet93** [comment, marked as answer]
@jcheese1 I think there is using `<ConnectButton.Custom>` and its render props.
Although it would be awesome to have access to this method outside of render part, using some custom hook or sth.
I asked about it here:
https://github.com/rainbow-me/rainbowkit/discussions/324
